### ISSUE TITLE
message.content is undefined, "that file might've been a virus"

### ISSUE THREAD

**Issue by emberglazee** [forscht/ddrive#40]
i was starting my ddrive and i kept noticing that it kept crashing during launch. i've dug into this and found that discord, apparently, detected one of the uploading files as a virus and in result did not upload it. solved by deleting the messages, but i'd suggest adding a failsave for that
```js
// my own code i added to find the exact messages
// src/discordFS/index.js, just before messagesGroupByType
tempMessageCache.forEach(message => {
    if (!message.content) {
        debug('>>> INVALID MESSAGE CONTENT')                                              
        debug(message)
    }
})
```
![IMG_20221102_095141](https://user-images.githubusercontent.com/68788282/199420259-aa329bf1-c009-477f-b14d-1789d93c01be.jpg)
![IMG_20221102_095351](https://user-images.githubusercontent.com/68788282/199420505-8ee5427a-34c0-435d-8ee9-e450556f9574.jpg)

**Comment by forscht** [comment]
Thanks for bringing this up. Will patch it.

**Comment by forscht** [comment]
Fixed in version `3.7.1` #56
